Corona-Warn-App/src/main/assets/privacy_de.html

```diff
@@ -1264,6 +1268,44 @@ <h3>
     schließen kann. Weiteren Angaben über Ihre Nutzung der Corona-Warn-App erhält der Anbieter Ihres
     Betriebssystems jedoch nicht.
 </p>
+<h3>
+    o. Aktualisierungsfunktion für COVID-Zertifikate
+</h3>
+<p>
+    Es kann vorkommen, dass die App Ihnen die Aktualisierung eines gültigen digitalen
+    COVID-Zertifikats anbietet. Beschließen die EU-Mitgliedstaaten beispielsweise eine Änderung des
+    Inhalts oder der Datenstruktur der Daten im COVID-Zertifikat, kann es für die reibungslose
+    weitere Zertifikatsnutzung erforderlich sein, die jeweilige inhaltliche oder technische Änderung
+    auf ein vorhandenes digitales COVID-Zertifikat anzuwenden.
+</p>
+<p>
+    Dazu lädt die App im Hintergrund die aktuellen inhaltlichen und technischen Spezifikationen für
+    die digitalen COVID-Zertifikate vom Serversystem der App herunter. Die App überprüft dann anhand
+    der in den von Ihnen gespeicherten digitalen COVID-Zertifikaten enthaltenen Daten, ob eine
+    Aktualisierung eines gespeicherten digitalen COVID-Zertifikats über die App möglich ist. Sie
+    werden dann auf eine mögliche Aktualisierung hingewiesen. Für die Aktualisierung wird mit Ihrem
+    Einverständnis das zu aktualisierende digitale COVID-Zertifikat von der App verschlüsselt an das
+    Serversystem der App übermittelt. Wenn es sich bei dem zu aktualisierenden digitalen
+    COVID-Zertifikat um ein COVID-Impfzertifikat zum Nachweis einer Auffrischungsimpfung
+    (Booster-Impfung) handelt, werden dabei auch die digitalen COVID-Zertifikate zum Nachweis der
+    der Auffrischungsimpfung vorausgegangenen Grundimmunisierung bzw. Genesung an das Serversystem
+    der App übermittelt. Dort wird dann die Gültigkeit der übermittelten digitalen COVID-Zertifikate
+    geprüft (siehe Abschnitt 6.h.). Anhand der Daten aus den übermittelten digitalen
+    COVID-Zertifikaten wird eine neue Version des COVID-Impfzertifikats für die Auffrischungsimpfung
+    generiert. Bei der Aktualisierung werden die Daten in den übermittelten COVID-Zertifikaten
+    (siehe Abschnitte 5.h und 5.i) kurzzeitig verarbeitet. Das aktualisierte COVID-Zertifikat wird
+    sodann an Ihre App zurückübermittelt.
+</p>
+<p>
+    Um eine missbräuchliche mehrfache Verwendung der Aktualisierungsfunktion verhindern zu können,
+    wird die elektronische Signatur des aktualisierten digitalen COVID-Zertifikats in Form einer
```

Comment:
There is another point, that @timokoenig was stumbling upon:
Is it really true that the hash of the **updated** DCC is stored despite the hash of the DCC **to be updated**?
We wonder, if the statement is true, what would be the positive effect?
See also: https://github.com/Digitaler-Impfnachweis/certification-apis/issues/231#issuecomment-1056815201